### ISSUE TITLE
new wraps: xtl, xsimd, xtensor, xframe

### ISSUE THREAD

**Issue by jeandet** [mesonbuild/wrapweb#79]
Here is a bunch of wraps I made :)
- xtl : https://github.com/QuantStack/xtl  version 0.5.2
- xsimd : https://github.com/QuantStack/xsimd version 7.1.2
- xtensor : https://github.com/QuantStack/xtensor  http://quantstack.net/xtensor version 0.19.1
- xframe :  https://github.com/QuantStack/xframe version 0.1.0

**Comment by sarum9in** [comment]
xsimd and xtensor look pretty unique, xtl and xframe do not. What do you think the names should be?

**Comment by kyonifer** [comment]
Did these end up getting posted somewhere?

**Comment by sarum9in** [comment]
I can create repositories for them as `quantstack-$name` (e.g. `quantstack-xsimd`) if @jeandet has no objections.

**Comment by jeandet** [comment]
@sarum9in @kyonifer I have no time to work on this now, but you are welcome to have a look on my github:
https://github.com/jeandet/xtensor/tree/meson
https://github.com/jeandet/xtl/tree/meson
https://github.com/jeandet/xframe
https://github.com/jeandet/xsimd/tree/meson
They might be quite outdated now, I hope this helps.

@sarum9in quantstack-$name is ok for me.

**Comment by kyonifer** [comment]
Thanks for the links @jeandet, these look like a good starting point.

**Comment by kyonifer** [comment]
I ended up writing a zip/patch wrap for xtl/xtensor based on your git repo version @jeandet, which is hosted at https://github.com/kyonifer/meson-wraps/tree/master/patches. I ended up stripping some of the functionality since I just needed the wraps to depend on from other projects, and did not need tests and such to run. 

I also added wrappers for xtensor-julia and xtensor-python, which ended up being a bit of logic hunting for e.g. the Julia library.

Hosting them in my own repo is currently working for me, but if there's interest I can PR somewhere here to get them into wrapdb.
